### PR TITLE
Remove robustness presubmits for `amd64` arch

### PR DESCRIPTION
**Pull request by ArkaSaha30**
This PR will remove robustness presubmits for both `amd64` and `arm64` archs, periodics for `amd64` arch since it has been migrated to `kubernetes/test-infra` as prowjobs:
- https://github.com/kubernetes/test-infra/blob/master/config/jobs/etcd/etcd-periodics.yaml#L61-L230
- https://github.com/kubernetes/test-infra/blob/master/config/jobs/etcd/etcd-presubmits.yaml#L240-L279

Issue link: https://github.com/etcd-io/etcd/issues/18136

Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

**Comment by ArkaSaha30** [comment]
cc @ivanvc @serathius

File: .github/workflows/robustness.yaml [hunk 1]
```diff
@@ -3,16 +3,6 @@ name: Robustness
 on: [push, pull_request]
```
Comment (serathius) [inline .github/workflows/robustness.yaml:3]: Please remove both presubmits and non-arm nightly robustness test.

No benefit of having presubmit on arm64.

Comment (ArkaSaha30) [reply .github/workflows/robustness.yaml:3]: Sure 👍

**Review by ivanvc — APPROVED** [review]
LGTM. Thanks, @ArkaSaha30.

**Review by serathius — APPROVED** [review]

**Pull request merged**